app/video_routes.py

```diff
@@ -0,0 +1,144 @@
+from app import db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+from flask import Blueprint, request, jsonify
+import os 
+
+video_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+@video_bp.route("", methods = ["GET"])
+def get_videos():
+        videos = Video.query.all()
+        response_body = build_videos_response(videos)
+        return jsonify(response_body),200
```

Comment:
we can do these in one line:

```suggestion
        return jsonify(build_videos_response(videos)),200
```